docs/content/docs/tasks_usage/ingress.md

```diff
@@ -21,8 +21,10 @@ HTTPS ingress support is experimental. OSM supports one way TLS authentication t
 
 By default, OSM configures HTTP as the backend protocol for services when an ingress resource is applied with a backend service that belongs to the mesh. A mesh-wide configuration setting in OSM's `osm-config` ConfigMap enables configuring ingress with the backend protocol to be HTTPS. HTTPS ingress can be enabled by updating the `osm-config` ConfigMap in `osm-controller`'s namespace (`osm-system` by default).
 
-Patch the ConfigMap by setting `use_https_ingress: "true"`.
+Patch the ConfigMap by setting use_https_ingress: "true".
+> Note: To make this change persistent between upgrades, see osm mesh upgrade --help.
```

Comment:
Why is this note not in egress.md? Should we put this in a generic place.

Reply:
The note was only added to places where `kubectl patch` is used. The egress doc was updated to use `osm mesh upgrade`, so the note seemed less relevant to me there.

Reply:
Was L28 in this file missed out?

Reply:
We used `kubectl patch` when `osm install` wasn't previously mentioned in the doc, Jon explains more [above](https://github.com/openservicemesh/osm/pull/2765#discussion_r589660727)

Reply:
This seems inconsistent with other usages of `osm mesh upgrade` and needs to be explicitly documented.